validator/main.go

```diff
@@ -8,38 +8,53 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/aws/amazon-cloudwatch-agent-test/internal/awsservice"
 	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/restart"
 	"github.com/aws/amazon-cloudwatch-agent-test/validator/models"
 	"github.com/aws/amazon-cloudwatch-agent-test/validator/validators"
 )
 
 var (
 	configPath      = flag.String("validator-config", "", "A yaml depicts test information")
 	preparationMode = flag.Bool("preparation-mode", false, "Prepare all the resources for the validation (e.g set up config) ")
+	runTest         = flag.String("test-name", "", "Test name to execute")
 )
 
 func main() {
 	flag.Parse()
 
 	startTime := time.Now()
-	vConfig, err := models.NewValidateConfig(*configPath)
-	if err != nil {
-		log.Fatalf("Failed to create validation config : %v \n", err)
-	}
 
-	if *preparationMode {
-		if err = prepare(vConfig); err != nil {
-			log.Fatalf("Prepare for validation failed: %v \n", err)
+	// validator calls test code to get around OOM issue on windows hosts while running go test
```

Comment:
Increasing the instance type was the first thing I tried. It didn't seem tied to instance size but more like something else that is unknown (it could be terraform ssh or something else).  The best workaround I found was to extend `validator`.